jsapp/js/main.es6

```diff
@@ -86,22 +85,10 @@ if (document.head.querySelector('meta[name=kpi-root-path]')) {
     Modal.setAppElement('#kpiapp');
     return $d.get(0);
   })();
-
-  render(<AllRoutes />, el);
-
-  if (module.hot) {
-    module.hot.accept('js/app', () => {
-      const AllRoutes = require('js/app').default;
-      render(
-        <AppContainer>
-          <AllRoutes />
-        </AppContainer>,
-        el
-      );
-    });
-  }
+  const root = createRoot(el);
+  root.render(<AllRoutes />);
```

Comment:
- Switch to the React 17 renderer with `createRoot`, `root.render`
- Don't need `module.hot` and the `react-hot-loader` app container, if we're using react refresh.